src/hackerspace_online/tests/test_auth.py

```diff
@@ -1,17 +1,71 @@
 import re
+from unittest.mock import patch
 
 from django.contrib.auth import get_user_model
 from django.core import mail
 from django.shortcuts import reverse
 
 from django_tenants.test.cases import TenantTestCase
 from django_tenants.test.client import TenantClient
+from django_tenants.utils import get_public_schema_name
 
 from hackerspace_online.tests.utils import ViewTestUtilsMixin
 
 User = get_user_model()
 
 
+class NonPublicOnlyAuthViewsTest(ViewTestUtilsMixin, TenantTestCase):
+    """
+    Custom `non_public_only_view` decorator was applied on every `allauth` views.
+    """
+
+    def setUp(self):
+        # Every test needs access to the request factory.
+        # https://docs.djangoproject.com/en/3.0/topics/testing/advanced/#the-request-factory
+        # self.factory = RequestFactory()
```

Comment:
I don't understand the purpose of these comments here.

Reply:
bad copy-paste